src/Compute/Compute/help/New-AzRestorePoint.md

```diff
@@ -13,9 +13,9 @@ This cmdlet can create a New Restore Point
 ## SYNTAX
 
 ```
-New-AzRestorePoint [-ResourceGroupName] <String> [-RestorePointCollectionName] <String> [-Name] <String>
- [[-DisksToExclude] <String[]>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
- [<CommonParameters>]
+New-AzRestorePoint [-ResourceGroupName] <String> -RestorePointCollectionName <String> -Name <String>
+ [-Location <String>] -RestorePointId <String> [-DisksToExclude <String[]>]
```

Comment:
please update markdown files to keep sync with code implementation. The syntax says RestorePointCollectionName is not a positional parameter now. Need be updated.